docs/projects/ad719x_asdz/index.rst

```diff
@@ -0,0 +1,173 @@
+.. _ad719x_asdz:
+
+AD719x EBZ/ASDZ HDL PROJECT
```

Comment:
`AD719x-ASDZ HDL project` should be the title, to match the name of the actual HDL project